<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OCC::IssuesWidget</class>
 <widget class="QWidget" name="OCC::IssuesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="_headerLabel">
     <property name="text">
      <string>List of issues</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QFormLayout" name="accountFolderLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="accountLabel">
         <property name="text">
          <string>Account</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="filterAccount">
         <item>
          <property name="text">
           <string>&lt;no filter&gt;</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="folderLabel">
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="filterFolder">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>&lt;no filter&gt;</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="1">
        <widget class="QCheckBox" name="showWarnings">
         <property name="text">
          <string>Show warnings</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="showIgnores">
         <property name="text">
          <string>Show ignored files</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="_treeWidget">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">4</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="copyIssuesButton">
       <property name="toolTip">
        <string>Copy the issues list to the clipboard.</string>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
